build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/ProcessLogcatTiming.cs

```diff
@@ -16,19 +16,47 @@ public class ProcessLogcatTiming : Task
 		[Required]
 		public string ApplicationPackageName { get; set; }
 
+		[Required]
+		public string DefinitionsFilename { get; set; }
+
 		public string ResultsFilename { get; set; }
 
+		Dictionary<string, Regex> definedRegexs = new Dictionary<string, Regex> ();
+		Dictionary<string, string> results = new Dictionary<string, string> ();
+
+		void LoadDefinitions ()
+		{
+			using (var reader = new StreamReader (DefinitionsFilename)) {
+				string line;
+
+				while ((line = reader.ReadLine ()) != null) {
+					int index = line.IndexOf (';');
+					if (index < 1 || index == line.Length)
```

Comment:
You should support comments within the file, and *probably* treat any line starting with `#` as a comment character.